YesterdayApi/Core/Email/IUserCredentials.cs

```diff
@@ -0,0 +1,8 @@
+namespace YesterdayApi.Core.Email
+{
+    public interface IUserCredentials
```

Comment:
UserCredentials gives off the vibe of password and username, maybe IUserEmailCredentials?

Reply:
What about an IEmailSender?

Reply:
I renamed it to: IEmailReciever

Reply:
Oh right, 👍